healthcheck_test.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestEndpoints(t *testing.T) {
+	t.Run("should return 200", func(t *testing.T) {
+		e := echo.New()
+
+		CreateHandler(e)
+
+		req := httptest.NewRequest(http.MethodGet, "/api/healthz", nil)
+		rec := httptest.NewRecorder()
+		e.ServeHTTP(rec, req)
+
+		assert.Equal(t, http.StatusOK, rec.Code)
```

Comment:
nitpick: you should assert against the response body here as well. You could also make sure the content-type header is present too if you're feeling crazy